test/test_testmon.py

```diff
@@ -192,6 +192,25 @@ def test_add():
         assert tuple(res) == (1, 0, 0), res
         sys.modules.pop('test_a', None)
 
+    def test_tlf(self, testdir):
+        tf = testdir.makepyfile(test_a="""
+            def test_add():
+                1/0
+        """, )
+        testdir.inline_run("--testmon", "-v")
+        sys.modules.pop('test_a', None)
+
+        result = testdir.runpytest("--testmon", "-v")
+        result.stdout.fnmatch_lines([
+            "*1 failed, 1 deselected*",
+        ])
+        tf.setmtime(1424880936)
```

Comment:
I'll leave it like that, as I don't think a comment would make it much better. I created an issue: https://github.com/tarpas/pytest-testmon/issues/72